nav2_controller/src/controller_server.cpp

```diff
@@ -464,7 +499,7 @@ void ControllerServer::computeAndPublishVelocity()
       nav_2d_utils::twist2Dto3D(twist),
       goal_checkers_[current_goal_checker_].get());
     last_valid_cmd_time_ = now();
-  } catch (nav2_core::PlannerException & e) {
+  } catch (nav2_core::NoValidControl & e) {
```

Comment:
I think this is sensible. The only other 2 conditions are TF and path invalid. For TF, that's usually from critically misconfigured robots that would be good to tell a user to "fix now" versus try to mask it. Path invalid is more... questionable, but I suppose the boundary conditions of the request won't change if the path is the reason for failure until a new path is given, so why even bother trying. 

Good call! 

I do wonder though about more generic throws due to "other" issues. We don't catch generic exceptions here - but if we did, we'd get all those other TF/plan/etc ones too, annoyingly. I suppose we could create a `GenericControllerError` from the base `ControllerException` and then capture the generic one, but maybe I'm over-optimizing for edge cases that don't exist yet.